pkg/model/api/v1/dashboard/variable_test.go

```diff
@@ -53,6 +55,31 @@ func TestUnmarshalJSONVariable(t *testing.T) {
 				},
 			},
 		},
+		{
+			title: "simple DatasourceVariable",
+			jason: `
+{
+  "kind": "DatasourceVariable",
+  "spec": {
+    "name": "datasourceVar",
+    "defaultValue": "prom-global",
+    "type": "PrometheusDatasource",
```

Comment:
Perhaps we could check at the service level that the datasource type selected is known in the different plugins. WDYT ?

Reply:
yup that's exactly what I meant.
Perhaps the most appropriate place would be here: https://github.com/perses/perses/blob/main/internal/api/impl/validate/validate.go#L73